oraclesrv/views.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+from flask import current_app, request, Blueprint, Response
+from flask_discoverer import advertise
+from requests.exceptions import HTTPError, ConnectionError
+
+import json
+
+from oraclesrv.utils import get_solr_data
+
+
+bp = Blueprint('oracle_service', __name__)
+
+def return_response(results, status_code):
+    """
+
+    :param results:
+    :param status_code:
+    :return:
+    """
+    r = Response(response=json.dumps(results), status=status_code)
+    r.headers['content-type'] = 'application/json'
+    return r
+
+def get_user_info_from_adsws(session):
+    """
+
+    :param session:
+    :return:
+    """
+    if session:
+        try:
```

Comment:
please consider propagating those execptions (i.e. not catching them); will something bad happen if we actually raise the error on failure? It might be easier to track down problems

Reply:
OK, propagating.